tmt/base.py

```diff
@@ -3331,6 +3345,13 @@ def show_runner(self, logger: tmt.log.Logger) -> None:
         for _, key_formatted, value_formatted in self.runner.facts.format():
             log(key_formatted, value_formatted, shift=1)
 
+    def prepare(self, tree: Tree) -> None:
```

Comment:
`prepare` alone is a bit overloaded, hinting to the reader as to what is the run getting ready for would be nice. `prepare_for_try`? `enter_try_session`?

Reply:
Makes sense, renamed in 93c1eec1.